File: hello.py
# Bug 1: Unused variable
unused_variable = 42

def main():
    print("Hello, world!")

# Bug 2: Division by zero
def divide_by_zero():
    result = 10 / 0  # This will cause a ZeroDivisionError

if __name__ == "__main__":
    main()
    divide_by_zero()

File: hello.java
public class BugExample {

    // Bug 1: Unused variable
    int unusedVariable = 42;

    public static void main(String[] args) {
        // Bug 2: Dead code
        if (false) {
            System.out.println("This code is never executed.");
        }

        // Bug 3: Null pointer exception
        String text = null;
        int length = text.length(); // This will throw a NullPointerException

        // Bug 4: Infinite loop
        while (true) {
            // This loop will never exit
        }
    }
}
